USAGE.md

```diff
@@ -167,3 +167,20 @@ For additional examples, check out existing manuscripts that use the Manubot (so
 + The Manubot 2018 Development Proposal ([source](https://github.com/greenelab/manufund-2018), [manuscript](https://greenelab.github.io/manufund-2018/))
 
 If you are using the Manubot, feel free to submit a pull request to add your manuscript to the list above.
+
+## Citing Manubot
+
+To cite the Manubot project or for more information on its design and history, see `@url:https://greenelab.github.io/meta-review/`:
+
+> **Open collaborative writing with Manubot**<br>
+Daniel S. Himmelstein, David R. Slochower, Venkat S. Malladi, Casey S.
+Greene, Anthony Gitter<br>
+_Manubot Preprint_ (2018) <https://greenelab.github.io/meta-review/>
+
+## Acknowledgments
+
+We would like to thank the contributors and funders whose support makes this project possible.
```

Comment:
I suggest changing `this project` to `the Manubot` project.  That should help avoid confusion in derived repositories.